diff --git a/python/regina/__init__.py b/python/regina/__init__.py
index 272c4b8d3..6e9c5dc9f 100644
--- a/python/regina/__init__.py
+++ b/python/regina/__init__.py
@@ -69,4 +69,15 @@ else:
 
+# In sageRegina, the census files are supplied differently.
+# Set the location here.
+try:
+    from .pyCensus import __path__ as _pyCensusPath
+    GlobalDirs.setDirs(
+        GlobalDirs.home(),
+        GlobalDirs.pythonModule(),
+        _pyCensusPath[0])
+except ImportError:
+    pass
+
 def reginaSetup(quiet = False, readline = True, banner = False,
                 snappyPath = True, namespace = None, builtinOpen = True):
     """
